cli/cmd/install.go

```diff
@@ -129,32 +131,16 @@ type (
 )
 
 const (
+	configStage       = "config"
+	controlPlaneStage = "control-plane"
+
 	prometheusImage                   = "prom/prometheus:v2.7.1"
 	prometheusProxyOutboundCapacity   = 10000
 	defaultControllerReplicas         = 1
 	defaultHAControllerReplicas       = 3
 	defaultIdentityTrustDomain        = "cluster.local"
 	defaultIdentityIssuanceLifetime   = 24 * time.Hour
 	defaultIdentityClockSkewAllowance = 20 * time.Second
-
-	nsTemplateName                  = "templates/namespace.yaml"
-	configTemplateName              = "templates/config.yaml"
-	identityConfigTemplateName      = "templates/identity-config.yaml"
-	identityTemplateName            = "templates/identity.yaml"
-	controllerConfigTemplateName    = "templates/controller-config.yaml"
-	controllerTemplateName          = "templates/controller.yaml"
-	webConfigTemplateName           = "templates/web-config.yaml"
-	webTemplateName                 = "templates/web.yaml"
-	prometheusConfigTemplateName    = "templates/prometheus-config.yaml"
-	prometheusTemplateName          = "templates/prometheus.yaml"
-	grafanaConfigTemplateName       = "templates/grafana-config.yaml"
-	grafanaTemplateName             = "templates/grafana.yaml"
-	resourcesTemplateName           = "templates/_resources.yaml"
-	serviceprofileTemplateName      = "templates/serviceprofile.yaml"
-	proxyInjectorConfigTemplateName = "templates/proxy_injector-config.yaml"
-	proxyInjectorTemplateName       = "templates/proxy_injector.yaml"
-	spValidatorConfigTemplateName   = "templates/sp_validator-config.yaml"
-	spValidatorTemplateName         = "templates/sp_validator.yaml"
```

Comment:
Since these strings only occur in one place, I prefer to just hard-code them. This way I can look at one place in the file rather than two to get my head around what's happening.